docs/configmanager.md

```diff
@@ -38,3 +39,13 @@ configManager:
   ocmBaseUrl: https://api.openshift.com
   watchInterval: 60
 ```
+
+### LOCAL UpgradeConfig Manager
+
+The following configuration fields must be set:
+
+| Field | Description | Example |
+| --- | --- | --- |
+| `source` | Indicates the type of config manager being used | `LOCAL` |
+| `LocalConfigName` | Name of the Local config being used | `example-upgrade-config` |
```

Comment:
@mrbarge was the config a singleton? "managed-upgrade-config" ?

Reply:
@dofinn Thanks for review. Yes it seems [upgradeconfig CR name](https://github.com/openshift/managed-upgrade-operator/blob/3518984ff470946edfb8156f1ac388c4c0be98ec/pkg/localprovider/localprovider.go#L14) (managed-upgrade-config) is fixed which is looked by MUO.

https://github.com/openshift/managed-upgrade-operator/blob/master/pkg/localprovider/localprovider.go#L63-L80